config/generator-templates.json

```diff
@@ -5,5 +5,8 @@
   { "value": "@asyncapi/nodejs-ws-template", "text": "Node.js WebSockets" },
   { "value": "@asyncapi/java-spring-cloud-stream-template", "text": "Java Spring Cloud Stream" },
   { "value": "@asyncapi/java-spring-template", "text": "Java Spring Boot" },
-  { "value": "@asyncapi/python-paho-template", "text": "Python Paho" }
-]
\ No newline at end of file
+  { "value": "@asyncapi/python-paho-template", "text": "Python Paho" },
+  { "value": "@asyncapi/go-watermill-template", "text": "Go Watermill" },
+  { "value": "@asyncapi/dotnet-rabbitmq-template", "text": ".NET RabbitMQ" },
+  { "value": "@asyncapi/ts-nats-template", "text": "TypeScript NATS" }
```

Comment:
https://github.com/asyncapi/ts-nats-template is archived for very long, please remove